src/runtime_src/core/include/xrt.h

```diff
@@ -120,7 +143,7 @@ struct xclDeviceInfo2 {
   unsigned short mMgtVtt;
   unsigned short m1v2Bottom;
   unsigned long long mDriverVersion;
-  unsigned mPciSlot;
+  unsigned int mPciSlot;
```

Comment:
Instead of "int" should this value be a long for on linux an int is 64bits and windows it is 32bits.

Reply:
Ok. I see what you are saying.   This struct is not well designed by using 'int'.  Should be explicit about size.